src/doc/src/reference/unstable.md

```diff
@@ -378,6 +379,21 @@ It's possible to update `my-dependency` to a pre-release with `update -Zunstable
 This is because `0.1.2-pre.0` is considered compatible with `0.1.1`.
 It would not be possible to upgrade to `0.2.0-pre.0` from `0.1.1` in the same way.
 
+## update-breaking
+
+* Tracking Issue: [#12425](https://github.com/rust-lang/cargo/issues/12425)
+
+The `update-breaking` feature allows upgrading dependencies to breaking versions
```

Comment:
Not quite correct to refer to this as `update-breaking` (backticks included) as a feature name as that implies its `-Zupdate-breaking`  when this is `-Zunstable-options`.  We have other examples of `-Zunstable-options` here, like `--out-dir`, that you can look to as examples

Reply:
Fixed.